SCPDiscordPlugin/Languages/zh-Hans.yml

```diff
@@ -0,0 +1,1419 @@
+  "GunE11SR":"Epsilon-11标准步枪",
+  "GunCrossvec":"Crossvec冲锋枪",
+  "Ammo556x45":"5.56mm子弹",
+  "GunFSP9":"FSP9",
+  "GunLogicer":"轻机枪",
+  "GrenadeHE":"手榴弹",
+  "GrenadeFlash":"闪光弹",
+  "Ammo44cal":"44号子弹",
+  "Ammo762x39":"7.62mm子弹",
+  "Ammo9x19":"9mm子弹",
+  "GunCOM18":"COM-18",
+  "SCP018":"SCP-018",
+  "SCP268":"SCP-268",
+  "Adrenaline":"肾上腺素",
+  "Painkillers":"止痛药",
+  "Coin":"硬币",
+  "ArmorLight":"轻型护甲",
+  "ArmorCombat":"战术护甲",
+  "ArmorHeavy":"重型护甲",
+  "GunRevolver":"左轮手枪",
+  "GunAK":"AK步枪",
+  "GunShotgun":"散弹枪",
+  "SCP330":"SCP-330",
+  "SCP2176":"SCP-2176",
+  "SCP244a":"SCP-244-A",
+  "SCP244b":"SCP-244-B",
+  "SCP1853":"SCP-1853",
+  "ParticleDisruptor":"3-X离子分裂者",
+  "GunCom45":"COM-45",
+  "SCP1576":"SCP-1576",
+  "Coal":"煤块",
+  "Scp2536_2":"SCP-2536-2",
+  "Jailbird":"囚鸟",
+
+  # SCP-914 模式
+  "Rough":"超粗加工",
+  "Coarse":"粗加工",
+  "OneToOne":"1:1",
+  "VeryFine":"精加工",
+  "Fine":"超精加工",
+
+  # 钥匙卡权限
+  #"None":"无",
+  #"Checkpoints":"检查点权限",
+  "ExitGates":"出口大门权限",
+  #"Intercom":"广播室权限",
+  "AlphaWarhead":"Alpha核弹权限",
+  "ContainmentLevelOne":"收容1级权限",
+  "ContainmentLevelTwo":"收容2级权限",
+  "ContainmentLevelThree":"收容3级权限",
+  "ArmoryLevelOne":"武器1级权限",
+  "ArmoryLevelTwo":"武器2级权限",
+  "ArmoryLevelThree":"武器3级权限",
+  "ScpOverride":"SCP-override",
+
+  # 某些门的设置
+  "PortallessBreakableDoor":"可破坏门",
+  "BreakableDoor":"不可破坏门",
+  "door\\(Clone\\)":"克隆门"
+]
+
+# 对最终版本的消息进行处理，这是唯一影响玩家名称的正则表达式。
+final_regex: ["discord\\.gg":"","http://":"","https://":"","www\\.":"", "<@":"<", "<@!":"<!"]
+
+messages:
+  onroundstart:
+    message: "**回合已经开始**"
+    regex: []
+
+  onconnect:
+    # ipaddress - 字符串(string)
+    # steamid - 字符串(string)
+    # jointype - CentralAuthPreauthFlags
+    # region - 字符串(string)
+    message: "玩家 (Steam64位ID <var:steamid> IP <var.ipaddress>)正在尝试连接..."
+    regex: []
+
+  onplayerleave:
+    # ipaddress - 字符串(string)
+    # playerid - 数字(int)
+    # steamid - 字符串(string)
+    # name - 字符串(string)
+    message: "**玩家 <var:name> (<var:steamid> <var.ipaddress>) 离开了服务器**"
+    regex: []
+
+  onroundend:
+    # 回合时长(duration) - int
+    # 胜利阵营(leadingteam) -string
+    # dclassalive - int
+    # dclassdead - int
+    # dclassescaped - int
+    # dclassstart - int
+    # mtfalive - int
+    # 基金会死亡(mtfdead) - int
+    # mtfstart - int
+    # scientistsalive - int
+    # scientistsdead - int
+    # scientistsescaped - int
+    # scientistsstart - int
+    # scpalive - int
+    # SCP死亡数(scpdead) - int
+    # SCP击杀(scpkills) - int
+    # scpstart - int
+    # warheaddetonated - bool
+    # warheadkills - int
+    # zombiesalive - int
+    # zombieschanged - int
+    message: >+
+      **回合结束！ 回合时长 <var:duration> 分钟**
+      ```
+      逃脱的D级人员数:            <var:dclassescaped>/<var:dclassstart>\n
+      获救的科学家数:         <var:scientistsescaped>/<var:scientistsstart>\n
+      剩余SCP:             <var:scpdead>/<var:scpstart>\n
+      SCP击杀数:              <var:scpkills>\n
+      Alpha核弹 <var:warheaddetonated> 引爆.
```

Comment:
How does this look in discord? If possible these columns should be aligned like in the english version.

![image](https://github.com/KarlOfDuty/SCPDiscord/assets/18099825/a2906255-4399-4f8a-a76e-bd0207857936)

Reply:
The issue has been modified
Now it's all the same